functions/Get-DbaTrigger.ps1

```diff
@@ -0,0 +1,117 @@
+﻿Function Get-DbaTrigger
+{
+<#
+.SYNOPSIS
+Get all existing triggers on one or more SQL instances.
+
+.DESCRIPTION
+Get all existing triggers on one or more SQL instances.
+
+Default output includes columns ComputerName, SqlInstance, Database, TriggerName, IsEnabled and DateLastMofied.
+
+.PARAMETER SqlInstance
+The SQL Instance that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user.
+
+.NOTES
+Author: Klaas Vandenberghe ( @PowerDBAKlaas )
+
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+.LINK
+ https://dbatools.io/Get-DbaTrigger
+
+.EXAMPLE
+Get-DbaTrigger -SqlInstance ComputerA\sql987
+
+Returns a custom object displaying ComputerName, SqlInstance, Database, TriggerName, IsEnabled and DateLastMofied.
+
+.EXAMPLE
+Get-DbaTrigger -SqlInstance 'ComputerA\sql987','ComputerB'

```

Comment:
Please keep using SqlServer in examples until 1.0. It will be easy to do mass replaces.